src/main/java/com/blackducksoftware/integration/jira/issue/handler/JiraIssuePropertyWrapper.java

```diff
@@ -89,76 +92,84 @@ public void addIssuePropertyJson(final Long issueId, final ApplicationUser user,
+    public List<EntityProperty> findProperties(String queryString) {
         if (queryString == null) {
             return Arrays.asList();
         }
         logger.debug("Querying for property: " + queryString);
-        final EntityPropertyQuery<?> query = jsonEntityPropertyManager.query();
-        final EntityPropertyQuery.ExecutableQuery executableQuery = query.key(queryString);
-        final List<EntityProperty> props = executableQuery.find();
+        EntityPropertyQuery<?> query = jsonEntityPropertyManager.query();
+        EntityPropertyQuery.ExecutableQuery executableQuery = query.key(queryString);
+        List<EntityProperty> props = executableQuery.find();
         return props;
     }
 
-    public List<IssueProperties> findIssuePropertiesByBomComponentUri(final String bomComponentUri) throws JiraIssueException {
+    public List<IssueProperties> findIssuePropertiesByBomComponentUri(String bomComponentUri) throws JiraIssueException {
         logger.debug("Find issue by Bom Component URI: " + bomComponentUri);
-        final List<IssueProperties> foundProperties = new ArrayList<>();
+        List<IssueProperties> foundProperties = new ArrayList<>();
 
-        final List<EntityProperty> properties = findProperties(bomComponentUri);
-        for (final EntityProperty property : properties) {
-            final IssueProperties issueProperties = createIssuePropertiesFromJson(property.getValue());
+        List<EntityProperty> properties = findProperties(bomComponentUri);
+        for (EntityProperty property : properties) {
+            IssueProperties issueProperties = createIssuePropertiesFromJson(property.getValue());
             logger.debug("findIssuesByBomComponentUri(): propertyValue (converted from JSON): " + issueProperties);
             foundProperties.add(issueProperties);
         }
         return foundProperties;
     }
 
-    public void addIssueProperties(final Long issueId, final ApplicationUser user, final String key, final IssueProperties propertiesObject) throws JiraIssueException {
+    public void addIssueProperties(Long issueId, ApplicationUser user, String key, IssueProperties propertiesObject) throws JiraIssueException {
         String jsonValue = "";
         if (null != propertiesObject) {
             jsonValue = gson.toJson(propertiesObject);
         }
         addIssuePropertyJson(issueId, user, key, jsonValue);
     }
 
-    public void addProjectProperty(final Long issueId, ApplicationUser user, final String key, final Object value) throws JiraIssueException {
+    public void addAlertIssueProperties(Long issueId, ApplicationUser user, AlertIssueSearchProperties propertiesObject) throws JiraIssueException {
+        String jsonValue = "";
+        if (null != propertiesObject) {
+            jsonValue = gson.toJson(propertiesObject);
+        }
+        addIssuePropertyJson(issueId, user, ALERT_PROPERTY_KEY, jsonValue);
+    }
```

Comment:
These are the important changes to this file. The other changes should be removing "final"